src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  /* font-weight: 300; */
+  color: #3d4e61;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  background-color: #fff;
+  font-weight: bold;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search::placeholder {
+  font-weight: normal;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f6 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.large > .search {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.small > .search {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.large > .icon {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  width: 19px;
+  height: 19px;
+  display: inline-block;
+}
+
+.small > .icon {
+  position: absolute;
+  left: 13px;
+  top: 15px;
+  width: 11px;
+  height: 11px;
+  display: inline-block;
+}
```

Comment:
don't increase selectors specificity, remove `>`. You can deal with it without `>`.